slack/web/client.py

```diff
@@ -285,6 +296,19 @@ def admin_users_setAdmin(
         kwargs.update({"team_id": team_id, "user_id": user_id})
         return self.api_call("admin.users.setAdmin", json=kwargs)
 
+    def admin_users_setExpiration(
+        self, *, expiration_ts: str, team_id: str, user_id: str, **kwargs
```

Comment:
> expiration_ts: str

This could be `int` type instead. The value is not the `ts` specific to Slack but an epoch time (seconds).

Reply:
I wasn't sure if I should make it `int` or `string` when I added this method. So I looked at the types of other `*_ts` arguments in the file and saw that they were using `str`. I'm not opposed to changing these to `int`, but we should probably file a separate issue for it so we can change them all at once if that is the case. https://github.com/slackapi/python-slackclient/blob/master/slack/web/client.py#L513

Reply:
I checked out the [actual method definition](https://slack-github.com/slack/webapp/blob/master/api_metadata/methods/admin.users/setExpiration.api#L40), and you are correct that it should be integer. I'll fix it for my PR and file an issue to review the other `_ts` used throughout the file.